src/styles/SectionHero.less

```diff
@@ -0,0 +1,23 @@
+.hero {
+    color: #333;
```

Comment:
Use color variables in separate file?

Reply:
Legger til color.less nå slik det er gjort i repoet til rodekors.